### ISSUE TITLE
[Feature request] Install multiple PKG from command line

### ISSUE THREAD

**Issue by nixxou** [RPCS3/rpcs3#14398]
### Quick summary

Right now, the option, --installpkg in the command line only allow to install a single pkg
It would be nice if we could put a folder as argument an get the UI that allow to install multiple pkg

![image](https://github.com/RPCS3/rpcs3/assets/45721836/3249e44f-a0de-4456-8d0a-bc3899e2644c)

**Comment by Megamouse** [comment]
I don't understand why you want a CLI option only to use the UI anyway.
Just use the UI then

**Comment by nixxou** [comment]
I would prefer a way to install pkg from command line without gui, but since the installpkg option pop up gui, i just tried to ask for something simple.

As for the usecase, i usually run games directly from command line, and i don't go to the UI much and i would like a way to ask to auto ask for pkg install if it's the first run of the game (you have pkg but no title id folder exist on game). PKGDIR folder don't seems to be an option because it wont trigger the pkg install ui if you run the game from command line pointing to the eboot directly and there don't seems to be a command line equivalent of the "Open" option as command line.

TLDR : Niche usecase, so i just try to ask to a tiny update over the already existing --installpkg command.
